main.py

```diff
@@ -33,6 +32,21 @@ def initialize_players(number_of_players, deck):
     return players
 
 
+def find_most_common_color(cards):
+    cards_colors = [card.color for card in cards if card.color != "change color"]
+    most_common_color = max(set(cards_colors), key=cards_colors.count)
+    return most_common_color
+
+
+def put_card_on_table(deck):
```

Comment:
Very good function! 
the name confused me a little bit because I was not sure who put the card on the table, the player of the first time for the initializing. 
Try to think on a more explicit name.